tests/unit/components/proxy.spec.js

```diff
@@ -20,7 +20,9 @@ describe('DocsProxy', () => {
         value: NextProxy,
         next
       },
-      component: Component
+      fixture: { component: Component },
+      onComponentRef: () => console.log('on component ref'),
```

Comment:
Since we're not using these in the tests I would make them empty functions to better express that.